sdks/java/core/pom.xml

```diff
@@ -121,6 +121,13 @@
         </executions>
       </plugin>
 
+      <!-- Ensure that the Maven jar plugin runs before the Maven
+        shade plugin by listing the plugin higher within the file. -->
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
```

Comment:
It might, but maven docs only say that the execution order of plugins is based first on phase order and then within a single phase it depends on the order in which they are defined in `plugns`. It doesn't say anything about `pluginManagement` affecting order. It also says that the order is inherited from parent modules but it is unclear as to what happens if I declare conflicting orders within a child module or just redefine the same plugin within a child module.